packages/teleport/src/Apps/Apps.tsx

```diff
@@ -73,10 +75,23 @@ export function Apps(props: State) {
           isEnterprise={isEnterprise}
           clusterId={clusterId}
           canCreate={canCreate}
-          onCreate={showAddApp}
+          onClick={showAddApp}
+          emptyStateInfo={emptyStateInfo}
         />
       )}
       {isAddAppVisible && <AddApp onClose={hideAddApp} />}
     </FeatureBox>
   );
 }
+
+const emptyStateInfo: EmptyStateInfo = {
+  title: 'ADD YOUR FIRST APPLICATION',
+  description:
+    'Teleport Application Access provides secure access to internal applications without the need for a VPN but with the audibility and control of Teleport.',
```

Comment:
Quick access to web applications running behind NAT and firewalls with security and compliance. Follow [the documentation](https://goteleport.com/docs/application-access/getting-started/) to get started.